tests/Template/ClassTemplateExtendsTest.php

```diff
@@ -4545,6 +4545,8 @@ class Secondary {}
                     '$c3' => 'c3<Secondary, RealE>',
                     '$resultC3' => 'RealE|Secondary',
                 ],
+                'ignored_issues' => [],
+                'php_version' => '8.0',
```

Comment:
oh, weird, I'd have expected promoted properties to be flagged when not in PHP 8...

Reply:
I didn't look into what this specific test was testing, TBH - I just added PHP 8 because it has a native union type hint.

Reply:
yeah, I was checking why this was necessary and I saw promoted properties too, it was just a remark :p